### PR TITLE
Sf waitlist

### PR DESCRIPTION
**Pull request by TMcMeans**
Added the static SF page and styling. Paired with Craig on fancy css maneuvering for adding the autoplay video in the hero. Take a look!

### CODE REVIEW

**Comment by TMcMeans** [comment]
As per the last commit, I wound up having to remove the temporary test.js file in the root and the update() method from sheets.js since we wound up pivoting away from using google spreadsheets to save user data from the waitlist page. We are sending user data to Cliniko now.

File: apps/marketing/components/SFWaitlist/index.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import { GraphQLClient } from 'graphql-request'
+
+const cx = classNames.bind(styles)
+
+const SFWaitlist = () => {
+  const nameInput = useRef()
+  const emailInput = useRef()
+
+  const handler = () => {
+    Router.push({
```
Comment (craigspaeth) [inline apps/marketing/components/SFWaitlist/index.js]: 👍

File: apps/marketing/components/SFWaitlist/index.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+              }
+            }
+            `,
+            {
+              input: {
+                name: nameInput.current.value,
+                email: emailInput.current.value
+                // phone: personPhone,
+                // isReferral: true,
+                // referralNote: {
```
Comment (craigspaeth) [inline apps/marketing/components/SFWaitlist/index.js]: Not a blocker for this PR so I made a note in Trello—we should make sure to include that again as it'll help the Marketing team find who came through this page.

**Comment by craigspaeth** [comment]
👏 Well done! This is an excellent start and just a little bit more QA/wiring up to go that I logged in the ticket. Going to merge this anyways b/c it's not a public page and the code looks great!

**Comment by lionsdwarf** [comment]
This looks great, nice work @TMcMeans ! 👏

**Pull request merged**